single-line-diagram-force-layout/src/main/java/com/powsybl/sld/force/layout/ForceSubstationLayout.java

```diff
@@ -8,13 +8,8 @@
 
 import com.powsybl.sld.layout.*;
```

Comment:
I think the whole file has to be moved out to `single-line-diagram-core` artifact, together with its factory. Indeed the force-layout algorithm could be of interest for other repos which are not related to single-line-diagram.